client.js

```diff
@@ -126,16 +140,16 @@ function getState() {
 function recorderProcess(e) {
     var samp = e.inputBuffer.getChannelData(0)
     state = getState()
-    if(state == 'gray') return
+    if(state === 'gray') return
     for(var i = 0; i < e.inputBuffer.length; i++) {
-        if(Math.abs(samp[i]) > 0.1) {
-            if(state == 'recording') {
+        if(Math.abs(samp[i]) > 0.4) { // volum audio
```

Comment:
E o chestie relativa ce difera de la microfon la microfon. De vreme ce nu se face o calibrare inainte, nu ai cum sa-ti dai seama ce volum e mai protrivit. In cazul meu, cu 10% imi inregistra respiratia si sunetul tastelor...